samples/GalleryApp/MainPage.xaml.cs

```diff
@@ -21,29 +16,81 @@ namespace GalleryApp
     /// </summary>
     public sealed partial class MainPage : Page
     {
+        public ObservableCollection<SkillView> SkillPages { get; } = new ObservableCollection<SkillView>();
+
+        private static string m_filePath = "\\Pages\\SkillViewGlossary.json";
```

Comment:
>m_filePath [](start = 30, length = 10)

when the json is copied over to the bin folder during build, does the Pages folder get recreated?

Reply:
If the Pages does not exist, then a new folder will be created. Otherwise all files inside Pages folder gets overwrite with the latest version.

---
In reply to: [302169857](https://github.com/microsoft/WindowsVisionSkillsPreview/pull/45#discussion_r302169857) [](ancestors = 302169857)